docs/source/concept_models/vivarium_census_synthdata/concept_model.rst

```diff
@@ -1209,49 +1206,163 @@ visual inspection and quantitatively using an appropriate statistical
 test (would that be a Chi-Square test?).
 
 
-2.3.10 Component 14: Social Security Number
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+2.3.10 Component 14: Social Security Number (SSN) and Individual Taxpayer Identification Number (ITIN)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Eventually, this should be missing for some and not actually unique
+Background
+^^^^^^^^^^
+
+Ideally, SSN should be missing for some and not actually unique
 for others.  I need to do some research into how we represent this,
```

Comment:
Yes I agree, I think what you've done in the observers is a very nice solution. This text was from Abie's original version, perhaps I should remove this part since we are in agreement about this.